platform/server-config/src/main/java/org/motechproject/server/config/SettingsFacade.java

```diff
@@ -214,6 +220,36 @@ public void saveRawConfig(String filename, String jsonText) {
         }
     }
 
+    public String propertiesToJson(Properties props) {
+        Gson gson = new GsonBuilder().create();
+        Map<String, String> map = new HashMap<String, String>();
+        for (Entry<Object, Object> prop : props.entrySet()) {
+            map.put((String) prop.getKey(), (String) prop.getValue());
+        }
+        return gson.toJson(map);
+    }
+
+    public Properties getPropertiesFromRawConfigFile(String filename) {
+        String json = new String();
+        try {
+            json = IOUtils.toString(getRawConfig(filename));
+        } catch (Exception ex) {
+            return new Properties();
```

Comment:
1. Do not swallow and ignore the exception
2. Do not catch generic Exception, but specific one thrown in this block